### PR TITLE
update edit in place to 1.0.11

### PR DESCRIPTION
**Pull request by iFun**

### CODE REVIEW

File: polymer.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "npm": true,
+    "extraDependencies": [
```
Comment (AKobets) [inline polymer.json]: nit: extra whitespace before "extraDependencies"

**Comment by iFun** [comment]
> This change include lang terms? You will have to include that in the polymer package file. This would be in the BSI https://github.com/Brightspace/brightspace-integration/blob/master/polymer.json

do I have to include the extra dependencies in the BSI repo or this repo or both?

**Comment by m6jones** [comment]
Should be just the BSI sorry :( I didn't realize what repo I was in t ill I made the comment.

**Comment by iFun** [comment]
> Should be just the BSI sorry :( I didn't realize what repo I was in t ill I made the comment.

cool I was going to update BSI's PR as soon as this one is merged

**Pull request merged**